aws-athena/views/default-vw_pin_sale.sql

```diff
@@ -204,6 +204,29 @@ mydec_sales AS (
     not, so we'll exclude them to avoid afore mentioned duplicates. */
     WHERE num_cards_sale = 1
         OR (YEAR(mydec_date) > 2020)
+),
+
+max_version_flag AS (
+    SELECT
+        meta_sale_document_num,
+        MAX(version) as max_version
+    FROM {{ source('sale', 'flag') }}
+    GROUP BY meta_sale_document_num
+),
+
+sales_val AS (
+    SELECT
+        sf.meta_sale_document_num,
+        sf.sv_is_outlier,
+        sf.sv_is_ptax_outlier,
+        sf.sv_is_heuristic_outlier,
+        sf.sv_outlier_type,
+        sf.run_id AS sv_run_id,
+        sf.version AS sv_version
+    FROM {{ source('sale', 'flag') }} AS sf
+    JOIN max_version_flag AS mv
```

Comment:
Are we planning on allowing max_version to differ across document numbers?

Reply:
Yep. One doc number could have a max version of 1 whereas another could have a max version of 3. If we run the `manual_update.py` script, it allows us to subset the data we assign flags to which results in this difference.

Reply:
Rad.